ocs_ci/deployment/metallb.py

```diff
@@ -92,7 +97,9 @@ def catalog_source_created(self):
         )
 
     @retry(CommandFailed, tries=3, delay=15)
-    def create_catalog_source(self):
+    def create_catalog_source(
+        self, metallb_version=config.default_cluster_ctx.ENV_DATA.get("metallb_version")
```

Comment:
please remove this arg, update `config.ENV_DATA.get("metallb_version") `value before calling `create_catalog_source`